draft-ietf-quic-transport.md

```diff
@@ -4251,6 +4251,10 @@ arrives after many higher-numbered packets have been received.  An endpoint
 SHOULD use a large enough packet number encoding to allow the packet number to
 be recovered even if the packet arrives after packets that are sent afterwards.
 
+Prior to any acknowledgements being received for a packet number space, there is
+no largest acknowledge packet, so the full packet number is encoded in the
+header.
+
```

Comment:
I think that we need to move this paragraph up one.  And it might instead say:

> Prior to receiving an acknowledgement for a packet number space, the full packet number MUST be included.